#168211855 Create the Apps homepage

cvjude opened the pull request:
#### What does this PR do?
- Create the App home page

#### Description of the task to be completed
> Create the Call to Action
> Create the main cards
> Make the page functional
> Write tests

#### How can this be manually tested?
- Clone the repo
- Run the command below in your terminal to install all dependencies
```
npm install
``` 
- Run the command below to test
```
npm test
```

#### Any background context you want to provide?
none

#### What are the relevant Pivotal Tracker stories? 
- [#168211855](https://www.pivotaltracker.com/story/show/168211855)

obayomi96 reviewed (approved):
This is very cool @cvjude,  great work LGTM

chiboycalix commented on src/actions/Articles/index.js:
```diff
@@ -0,0 +1,49 @@
+    });
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.log(err);
```
I think you forgot to remove this console.log statement.

Hadeneekeh reviewed (approved):
LGTM!!!

chiboycalix reviewed (approved):
LGTM

cvjude committed:
feat(Homepage): Create home page

Create the Main article section
Modify the navBar to suit home page
Create the CTA
Write unit tests
Write end to end tests

[Delivers #168211855]

encodedBicoding reviewed (approved):
LGTM

Lundii reviewed (approved):
LGTM

cvjude committed:
Refactor Homepage styles
